<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>440</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Wespal</string>
  </property>
  <widget class="QWidget" name="grid">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="staWorkAreaParent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageWorkArea">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="hlPreviews">
          <item>
           <widget class="QScrollArea" name="previewOriginalContainer">
            <property name="whatsThis">
             <string>This is the original image's preview.</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="widgetResizable">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>270</width>
               <height>162</height>
              </rect>
             </property>
             <property name="whatsThis">
              <string/>
             </property>
             <widget class="ImageLabel" name="previewOriginal" native="true">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>72</width>
                <height>72</height>
               </rect>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="previewRcContainer">
            <property name="whatsThis">
             <string>This is a preview of a single transformation of the original image.</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="widgetResizable">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>271</width>
               <height>162</height>
              </rect>
             </property>
             <widget class="ImageLabel" name="previewRc" native="true">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>72</width>
                <height>72</height>
               </rect>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageNoFile">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblNoFile">
          <property name="text">
           <string>Open or drag an image here:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cmdOpen">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="panelMru">
          <layout class="QHBoxLayout" name="_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lblNoFile2">
               <property name="text">
                <string>Or select a recently opened image:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="listMru">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="hlCommands">
      <item>
       <widget class="QSlider" name="zoomSlider">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Zoom</string>
        </property>
        <property name="whatsThis">
         <string>Allows to choose the view's zoom level.</string>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>4</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" rowspan="3">
     <layout class="QHBoxLayout" name="hlOptions" stretch="1,0,1">
      <item>
       <spacer name="hpaddingLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlOptions" stretch="0,0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radRc">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis">
             <string>Selects color range-based (a.k.a. &quot;team color&quot;) recoloring.</string>
            </property>
            <property name="text">
             <string>Color &amp;range (TC)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radPal">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis">
             <string>Selects palette switching mode, to replace original colors with new ones using a color values list instead of ranges.</string>
            </property>
            <property name="text">
             <string>Palette &amp;switch</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="flKeyPal">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblKeyPal">
            <property name="text">
             <string>Palette &amp;key:</string>
            </property>
            <property name="buddy">
             <cstring>cbxKeyPal</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbxKeyPal">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="whatsThis">
             <string>Selects the original palette for transformation.</string>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QStackedWidget" name="staFunctionOpts">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="pageRcRanges">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="margin" stdset="0">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QListWidget" name="listRanges">
              <property name="whatsThis">
               <string>Allows to select color ranges to apply. Click on an item to preview the transformation to the right.</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pageRcPal">
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <property name="margin" stdset="0">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QFormLayout" name="flRcPal">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lblNewPal">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>New &amp;palette:</string>
                </property>
                <property name="buddy">
                 <cstring>cbxNewPal</cstring>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="cbxNewPal">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="whatsThis">
                 <string>You an choose the replacement palette in this box.</string>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QComboBox::AdjustToContents</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="hpaddingRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuMru">
     <property name="title">
      <string>Open &amp;Recent</string>
     </property>
     <addaction name="action_MruPlaceholder"/>
     <addaction name="separator"/>
     <addaction name="action_ClearMru"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="menuMru"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Reload"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Morning_Star"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuPreview_Background">
     <property name="title">
      <string>Preview &amp;Background</string>
     </property>
     <addaction name="actionPreviewBgDefault"/>
     <addaction name="actionPreviewBgWhite"/>
     <addaction name="actionPreviewBgLight"/>
     <addaction name="actionPreviewBgDark"/>
     <addaction name="actionPreviewBgBlack"/>
     <addaction name="separator"/>
     <addaction name="actionPreviewBgCustom"/>
    </widget>
    <addaction name="menuPreview_Background"/>
    <addaction name="separator"/>
    <addaction name="actionColor_ranges"/>
    <addaction name="action_Palettes"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_Morning_Star">
   <property name="text">
    <string>About &amp;Wespal...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionColor_ranges">
   <property name="text">
    <string>Custom Color &amp;Ranges...</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Palettes">
   <property name="text">
    <string>Custom &amp;Palettes...</string>
   </property>
  </action>
  <action name="action_Reload">
   <property name="text">
    <string>Reloa&amp;d</string>
   </property>
  </action>
  <action name="action_RecentPlaceholder">
   <property name="text">
    <string>&lt;Recent files&gt;</string>
   </property>
  </action>
  <action name="actionPreviewBgWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;White</string>
   </property>
  </action>
  <action name="actionPreviewBgLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Light</string>
   </property>
  </action>
  <action name="actionPreviewBgDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Da&amp;rk</string>
   </property>
  </action>
  <action name="actionPreviewBgBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Black</string>
   </property>
  </action>
  <action name="actionPreviewBgDefault">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Default</string>
   </property>
  </action>
  <action name="actionPreviewBgCustom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Custom...</string>
   </property>
  </action>
  <action name="action_MruPlaceholder">
   <property name="text">
    <string>&lt;Recent files&gt;</string>
   </property>
  </action>
  <action name="action_ClearMru">
   <property name="text">
    <string>&amp;Clear Menu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QWidget</extends>
   <header>src/imagelabel.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>previewOriginalContainer</tabstop>
  <tabstop>previewRcContainer</tabstop>
  <tabstop>radRc</tabstop>
  <tabstop>radPal</tabstop>
  <tabstop>cbxKeyPal</tabstop>
  <tabstop>listRanges</tabstop>
  <tabstop>cbxNewPal</tabstop>
  <tabstop>zoomSlider</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
